frontend/src/components/navigation/Header.tsx

```diff
@@ -1,7 +1,10 @@
-import React from "react";
-import { Button, Flex, Heading, Image } from "@chakra-ui/react";
+import React, { useContext, useState } from "react";
+import { Button, Flex, Heading, IconButton, Image } from "@chakra-ui/react";
+import { Icon } from "@chakra-ui/icon";
+import { BiUserCircle } from "react-icons/bi";
```

Comment:
not familiar with react-icons/bi, did you have to add any yarn packages for this?
if possible, lets stick to` react-icons/md`, you can find icons here https://react-icons.github.io/react-icons/icons?name=md